test/e2e/data/test-extension/deployment.yaml

```diff
@@ -0,0 +1,150 @@
+apiVersion: v1
```

Comment:
Accidental I assume?

Reply:
Yeah - this is still very much WIP - haven't come back to it since I figured out what was going on.

Reply:
> So if we go ahead we would also set it in the same places as fail-swap-on? (basically everywhere)

Yup - I think we should match up with what Kind is doing. We can probably do this ahead of time as these don't conflict AFAIK with the current way of setting the args - same as `fail-swap-on`